gaming/Makefile

```diff
@@ -0,0 +1,81 @@
+BUILD_DIR=$(PWD)/bin
+profile:
+	echo "Building profile service"
+	cd src/golang/profile-service && GOOS=linux GOARCH=386 go build -o ${BUILD_DIR}/profile-service main.go
```

Comment:
Does this mean that building/testing this sample only works on Linux-based systems? If so, it might be good to point that out in the README.

Reply:
For now, it does. I'll add it to README (which turns out was missing a BUILD section anyway!)